### DISCUSSION TITLE
OCR supported languages

### DISCUSSION THREAD

**Discussion by a7medo778** [jonaswinkler/paperless-ng#1564] [Support]
hey guys
i was wondering is arabic one of the supported languages via the OCR feature?
because there are many non-core apache tika files to enable that

**Comment by TheNomad11** [comment]
anyone knows that?

**Reply by bucketphobia** [reply]
@a7medo778 @TheNomad11 

Yes, it does support Arabic with a caveat - extracting text from digitally created PDFs is bugged in Arabic, and results in the text being displayed backwards. (bug from OCRmyPDF - not paperless)

The only solution I found was to force OCR, which works fine, but is very computationally heavy
